src/pywbem/tupleparse.py

```diff
@@ -1450,3 +1450,61 @@ def unpack_boolean(p):
         return None
     else:
         raise ParseError('invalid boolean %s' % p)
+
+
+def parse_value_instancewithpath_to_dict(tt):
+    """
+    Return parsed instancewithpath with name and value as a dict.
+
+    <!ELEMENT VALUE.INSTANCEWITHPATH ((CLASSPATH, CLASS) |
+                                    (INSTANCEPATH, INSTANCE))>
+    """
+
+    check_node(tt, 'VALUE.INSTANCEWITHPATH')
+
+    k = kids(tt)
+
+    if len(k) != 2:
+        raise ParseError('Expecting two elements, got %s' % k)
+
+    if name(k[0]) == 'CLASSPATH':
+        object = (parse_classpath(k[0]),
+                  parse_class(k[1]))
+    else:
+        path = parse_instancepath(k[0])
+        object = parse_instance(k[1])
+
+        object.path = path
+
+    return {name(tt): object}
+
+
+def parse_paramvalue_to_dict(tt):
```

Comment:
No need to use _to_dict suffix, just add a return type(rtype) to the docstring - https://thomas-cokelaer.info/tutorials/sphinx/docstring_python.html

Reply:
Also, explain what is tt in the context.